src/index.ts

```diff
@@ -1215,6 +1215,7 @@ async function claimLink({
 }
 
 async function claimLinkXChain(
```

Comment:
just for clarity I'd put isTestnet as the last argment (after destChainId)

Reply:
Followup question: any idea what the chainId for e.g. a cosmos chain looks like? @konradstrachan